src/app/actions.ts

```diff
@@ -35,45 +35,6 @@ export async function addToGroup({ email }: { email: string }) {
   }
 }
 
-export async function sendQuestion({
-  question,
-  slug,
-  recaptchaToken,
-  userId,
-}: {
-  question: string;
-  slug: string;
-  recaptchaToken: string;
-  userId: string;
-}) {
-  const isRecaptchaValid = await validateRecaptcha(recaptchaToken);
-  if (!isRecaptchaValid) {
-    console.error('Recaptcha validation failed');
-    return 400;
-  }
-  if (!question || !slug) {
-    return 400;
-  }
-  const res = await fetch(`https://konf-qna.kir-dev.hu/api/presentation/${slug}/question`, {
-    method: 'POST',
-    body: JSON.stringify({ content: question, userId }),
-  });
-  if (res.status === 200) {
-    return 201;
-  } else if (res.status === 400) {
-    return 400;
-  }
-  return 500;
-}
-
-async function validateRecaptcha(token: string) {
-  const res = await fetch('https://www.google.com/recaptcha/api/siteverify', {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/x-www-form-urlencoded',
-    },
-    body: `secret=${process.env.RECAPTCHA_SECRET}&response=${token}`,
-  });
-  const data = await res.json();
-  return data.success;
```

Comment:
It would've been better if this was left here as a comment, could be a big help next year.
Also at the network functions